Last-Update: 2014-08-10
Forwarded: not-needed
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: un-bundle liblz4 and link to the system one

--- a/setup.py
+++ b/setup.py
@@ -16,11 +16,11 @@
     packages=find_packages('src'),
     package_dir={'': 'src'},
     ext_modules=[
         Extension('lz4', [
-            'src/lz4.c',
-            'src/lz4hc.c',
             'src/python-lz4.c'
+        ], libraries=[
+            "lz4",
         ], extra_compile_args=[
             "-std=c99",
             "-O3",
             "-Wall",
--- a/src/python-lz4.c
+++ b/src/python-lz4.c
@@ -32,10 +32,10 @@
 #include <Python.h>
 #include <stdlib.h>
 #include <stdint.h>
 #include <math.h>
-#include "lz4.h"
-#include "lz4hc.h"
+#include <lz4.h>
+#include <lz4hc.h>
 #include "python-lz4.h"
 
 #define MAX(a, b)               ((a) > (b) ? (a) : (b))

